backend/app/tests/validators/test_pii_remover.py

```
@@ -0,0 +1,88 @@
+
+def test_validator_initialization(validator):
+    assert validator.language in ALL_SUPPORTED_LANGUAGES
+    assert isinstance(validator.entity_types, list)
+    assert len(validator.entity_types) > 0
+
+
+def test_validate_pass_result_when_presidio_returns_string(validator):
+    # Mock analyzer/anonymizer pipeline
+    with patch.object(validator, "run_english_presidio", return_value="no pii here"):
+        result = validator._validate("no pii here")
+        assert result.outcome is "pass"
```

Ruff (0.14.10)

Use `==` to compare constant literals

Replace `is` with `==`

(F632)